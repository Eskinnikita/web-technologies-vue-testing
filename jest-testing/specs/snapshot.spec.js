const user = {
    name: "Eskin Nikita",
    age: 21,
    job: "student"
}

test('user matches', () => {
    expect(user).toMatchSnapshot()
})