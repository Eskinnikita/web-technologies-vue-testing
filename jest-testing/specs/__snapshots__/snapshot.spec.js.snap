// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
Object {
  "age": 21,
  "job": "student",
  "name": "Eskin Nikita",
}
`;
